### ISSUE TITLE
Voting category SVGIcons

### ISSUE THREAD

**Issue by local-minimum** [ludumdare/ludumdare#1282]
We have icons for all filters at the moment (though some could do with a bit of improvement), except for voting categories. And that looks bad. So we need some improvements. To look at the icons, they are in `http://192.168.48.48/dev_icons.php` of your dev environment.

## Voting
Overall (could use 'trophy')
Fun (???)
Innovation (could use 'lightbulb')
Theme (???)
Graphics (could use 'video-camera')
Audio (could use 'alert')
Humor (suggest finding/making a smiling mouth/face)
Mood (???)

## Categories
* The categories: 'All', 'Jam', 'Compo' could also have better unique icons but at least they have some.

## Featured
* I think 'Featured Event' could be using 'tag', but perhaps it be better with 'star'
* All events should use 'tags' not 'tag'.

## Sorting
The sortings 'smart' and 'classic' could both or at least the first one use 'lightbulb'.

## When we agree and have what we need: 
- [ ] the file `com/content-games/filter.js` needs updating.
- [ ] The opt-out list should also use the icons of the categories
- [ ] The trophies could use the new icons instead of all just the trophy-icon. So you get a golden light bulb for winning innovation.

**Comment by local-minimum** [comment]
@povrazor please update the post if you have ideas to keep it alive with what is decided. I don't care too much about my specific ideas for icons, but I do care for having them and improving the existing choices.

**Comment by povrazor** [comment]
I like this idea (especially the light bulb). That said, because of the number of original icons we already need, I would rather we come up with the best iconography for each category, than choosing ones that sorta fit (i.e. video camera for graphics).

**Comment by local-minimum** [comment]
One idea is to map categories to body-parts.

* Graphics: eye
* Audio: ear
* Mood: I would, knoeing biology want a brain here, but innovation would probably make more sense to have the brain. So heart. A real one since mood many times is dark and gritty.
* Humour: laughing mouth

Fun is hard to differentiate from humour. Also many times voting, but could be a hand, jusy because well crafted games that play well atr often more fun to play. And you play with hands most of the time.

That leaves overall, which if forced to the body eould be a non-gendered silhuette. But this is also a special category so it would make sense having it dissimilar from the others. So the current trophy would make sense.

And I skipped theme because I have no idea what that is. Small intestine? But also outside the body theme, I have mo real clue

**Comment by local-minimum** [comment]
If we want inanimate objects like the innovation-light bulb, then: 

* Audio could be a note or two, or an instrument like a saxophone. 
* Graphics could be a voxel or iconic Bob Ross.

The others are hard though.

**Comment by povrazor** [comment]
Just spitballing, but a Fedora hat for Mood is another option. The fedora is commonly associated with noir/detective stories, who are primarily known for moodyness. :tophat: (okay, that's a tophat, not a fedora, ha)

@Noojuno @sgstair @zwrawr if you have any ideas for catagory iconography, please share. :wink:

**Comment by sgstair** [comment]
* Overall: 🎰 (hehe) 👑  
* Fun: a seesaw comes to mind, as does confetti poppers 🎉 🎊 
* Theme: This is the hardest one I think. I'll have to think over it some more.
* Audio: 🎤 🎙  - I think a microphone might better represent the intent and reality of much of LD :)

**Comment by povrazor** [comment]
Yes, a crown :crown: is good for overall. 

In the same vein, Fun could be a balloon :balloon:

I think you're right about the microphone. It's more all encompassing of audio, and musical notes might not say that well enough.

Theme though... yeah. That's gonna take some brain cells to solve. :wink:

**Comment by povrazor** [comment]
I'm checking if the Twitterverse has any good ideas.

https://twitter.com/mikekasprzak/status/922535336034660352

**Comment by povrazor** [comment]
Actually I got a decent suggestion already: a seedling :seedling: 

Yes it's not completely obvious how that relates to theme, but the purpose of the theme is to get you started/inspire what you create.

**Comment by povrazor** [comment]
Some more interesting ideas, a key :key:. When flattened to 1 color, I think a key will be easier to identify than a seedling, but a seedling does make more sense.

Two more amusing suggestions, a Mario Bros style question block, and a box of Valentine chocolates (i.e. heart shaped). Really fun suggestions, but Question Marks would be confused with help icons, and anything heart shaped confused with Love/Karma.

That said, it brings up a good point that a game that's unusually popular could be given a heart trophy/icon, as a way of suggesting it's popular.

**Comment by local-minimum** [comment]
I like baloon too for fun. It is carefree up in the sky and doesn't confuse with humor.

To too keen on fedora for mood.

**Comment by local-minimum** [comment]
Mood sometimes depicted as cloud/stormy weather. Sometimes cloud is in place of brain in skull.

Though a seed is cool idea for theme it kind of confuses with innovation as seed indicate new, which is what innovations are

**Comment by local-minimum** [comment]
Color palette is sometimes an icon for theme, but hard with just 1bit image. Might also confuse with graphics.

**Comment by povrazor** [comment]
Yeah with icons it's not often there's going to be something perfect (Crown and Light Bulb do work pretty good). So it's a matter of picking the best of what's available.

**Comment by povrazor** [comment]
Overall: :crown:
Fun: :balloon:
Innovation: :bulb:
Theme: :seedling: or :key: or ??
Graphics: :eye: 
Audio: :microphone: 
Humor: :rofl: 
Mood: ![fedora32](https://user-images.githubusercontent.com/1570313/31916362-559d1572-b820-11e7-87c0-64fc5c5f1c61.png)

What's key with iconography is having a strong silhouette. The icons are potentially going to be very small, so they need to communicate an idea with the simplest of lines and shapes.

![hat4](https://user-images.githubusercontent.com/1570313/31916634-a82b3cfa-b821-11e7-94f2-0d2c6100e3d5.png) ![hat1](https://user-images.githubusercontent.com/1570313/31916624-9bc09c12-b821-11e7-8031-7b6754cbf31f.png) ![hat2](https://user-images.githubusercontent.com/1570313/31916626-9f562dec-b821-11e7-87db-1742b43eee6c.png) ![hat3](https://user-images.githubusercontent.com/1570313/31916629-a32b86ba-b821-11e7-846e-13f0cdca061c.png)

I'm somewhat partial to the fedora because there's really nothing else like it. Hats don't really suit any of the other categories. There's no real risk of confusion, as there is with the painter's palette :art:. It's just a question of what style (I prefer the 1st, but they're using transparent gradients in this icon, so how doable is TBD).

**Comment by local-minimum** [comment]
For some reason, the icon for humor just shows up as a solid black square for me. Which is a fun idea of an icon for humor.

Theme could be a frame, for a painting or a window partially opened. Because o how it frames the content but i only a starting point for what you fill it with.

The hat is as you put it second hat if you count crown. It is also quite western and male. Though not exlusvely. But at least in film as your mood reference.

**Comment by local-minimum** [comment]
I still think a heart (real, not the like) icon possibly with an EKG through it would give a strong reference to mood and emotions

**Comment by povrazor** [comment]
The problem is we have other heart :heart: iconography. i.e. post/comment love/karma. These icons need to stand out among other icon concepts we use (like hearts, stars, and rocket ships for publishing).

**Comment by local-minimum** [comment]
Not that heart, the real one ;)
